aiohttp_baseapi/project_template/src/.meta/packages

```diff
@@ -2,7 +2,7 @@
 --extra-index-url http://pypi.wargaming.net/root/prod/
```

Comment:
Could you please also remove these two wargaming-specific lines?

Reply:
Oh, yes of course.